docs/README.md

```diff
@@ -0,0 +1,57 @@
+# Set up HashBus for an event
+The purpose of this document is to describe how to set up HashBus to monitor tweets and display leaderboard.

```

Comment:
Not really. It's general documentation. We might want to consider whether this should be part of the root README.md.